brickstrap.sh

```diff
@@ -352,11 +373,11 @@ function brp_run_multistrap() {
     debug "ROOTDIR: ${ROOTDIR}"
     if [ ! -z ${FORCE} ] && [ -d "${ROOTDIR}" ]; then
         warn "Removing existing rootfs ${ROOTDIR}"
-        ${USER_UNSHARE} -- rm -rf ${ROOTDIR}
+        brp_unshare -- rm -rf ${ROOTDIR}
     fi
-    ${USER_UNSHARE} -- multistrap ${MSTRAP_SIM} --file "${MULTISTRAPCONF}" \
+    brp_unshare -- multistrap ${MSTRAP_SIM} --file "${MULTISTRAPCONF}" \
         --dir ${ROOTDIR} --no-auth
-    cp ${QEMU_STATIC} ${ROOTDIR}/usr/bin/
+    brp_setup_qemu_in_rootfs
```

Comment:
You didn't update the complementary statement in line 546 that excludes this file from the tar. Line 546 is still using the `QEMU_STATIC` variable.